django/db/models/sql/query.py

```diff
@@ -508,6 +512,14 @@ def has_results(self, using):
         compiler = q.get_compiler(using=using)
         return compiler.has_results()
 
+    def explain(self, using, format=None, **kwargs):
```

Comment:
Just a thought - we use `**kwargs` here, but should we use something else for clarity?

- `flags` - referred to as flags in the help documentation, but implication is they can only be booleans.
- `options` - as referred to in the [PostgreSQL documentation](https://www.postgresql.org/docs/current/static/sql-explain.html).
- `extra` - used by Django in the Expressions API.

Obviously the documentation would need updating to reflect this change...

Does Django have any convention here, @timgraham? I think one of the latter two makes sense.

Reply:
I like `options`, it seems to encompass both flags and... options? I'll change it to be this in the code as well as the docs